.github/workflows/build.yml

```
@@ -0,0 +1,89 @@
+
+          #      - name: wstunnel-builder-cache
+          #        id: docker_build_wstunnel-cache
+          #        uses: docker/build-push-action@v2
+          #        with:
+          #          push: true
+          #          tags: ghcr.io/erebe/wstunnel:builder-cache
+          #          target: builder-cache
+
+      - name: wstunnel
+        id: docker_build_wstunnel
+        uses: docker/build-push-action@v2
```

actionlint (1.7.4)

the runner of "docker/build-push-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)